### PR TITLE
Notifyre: bash: Fixes

### PR DESCRIPTION
**Pull request by a7r3**
Message is in Commit

**Need confirmation if things are working fine - Tested on Antergos KDE (Arch)**

Especially [this line](https://github.com/kaustubhhiware/NotiFyre/compare/master...a7r3:master#diff-f2e73e009bcaae5534d3638fbb66571dR57)

The Unicode character appears as a rectangle (indicating missing character), but it could just be me experiencing it.

### CODE REVIEW

File: notifyre.sh [hunk 1]
```diff
@@ -46,23 +40,21 @@ notif_timer() {
...
+        -i utilities-terminal \
+        -t 5000
     fi
 
-    if [ "$SOUND" -eq 1 ]; then
-      if [ "$elapsed" -gt "$SOUND_MIN" ]; then
-        paplay $ALERT
-      fi
+    if [ "$SOUND" -eq 1 ] && [ "$elapsed" -gt "$SOUND_MIN" ]; then
+        paplay $ALERT &> /dev/null || echo -en "\a";
```
Comment (a7r3) [inline notifyre.sh]: Want to confirm if `paplay` works properly, as well as its fallback

File: notifyre.sh [hunk 2]
```diff
@@ -46,23 +40,21 @@ notif_timer() {
...
-        -message "Completed in $elapsed seconds"\
-        -timeout 5\
+      terminal-notifier -title "NotiFyre $(echo $command_status)" \
+        -subtitle "Command : $commandx" \
+        -message "Completed in $elapsed seconds" \
+        -timeout 5 \
         -closeLabel "Gotcha!"
     else
       # Use notify-send for others
-      notify-send "NotiFyre $(echo $command_status)" "Terminal in ${p[-2]}/${p[-1]} and exit $? \$" \
```
Comment (kaustubhhiware) [inline notifyre.sh]: You forgot to add the first quote "Terminal in ${p[-2]}/${p[-1]} and exit $? \$"

**Comment by kaustubhhiware** [comment]
Thanks for your contribution! :smile: 

Please make the requested changes. 
Also, you should make a separate branch, and not from `master` itself. I'll ignore that for now.

File: notifyre.sh (hunk 1 above)
Comment (athityakumar) [reply notifyre.sh]: `paplay` should probably be used only while in non-Mac conditions.

File: notifyre.sh (hunk 2 above)
Comment (a7r3) [reply notifyre.sh]: I mentioned in the commit message, actually it isn't preferred
We need to have a small and precise notification, and that's why I removed that statement and the code relating to it

File: notifyre.sh (hunk 1 above)
Comment (a7r3) [reply notifyre.sh]: That's fine in that case.
Could you or anyone test if this works ? With a valid alert tone provided to `paplay`

Comment (kaustubhhiware) [reply notifyre.sh]: @a7r3 It works in bash,

@athityakumar Did not understand your point

Comment (a7r3) [reply notifyre.sh]: Thanks! @kaustubhhiware

File: notifyre.sh (hunk 2 above)
Comment (kaustubhhiware) [reply notifyre.sh]: The script is left pretty much open for configuration.

Let the message be there, if it does not suit someone, they can modify their local file.

Comment (a7r3) [reply notifyre.sh]: Well, okay.
I'm bringing back the message

File: notifyre.sh (hunk 1 above)
Comment (a7r3) [reply notifyre.sh]: @kaustubhhiware Probably he meant that `paplay` does not exist in a Mac System

Comment (athityakumar) [reply notifyre.sh]: @a7r3 - That's what I meant. 😄

File: notifyre.sh (hunk 2 above)
Comment (a7r3) [reply notifyre.sh]: @kaustubhhiware any emphasis on the '$' after `exit $?`
I couldn't analyze its presence

File: notifyre.sh (hunk 1 above)
Comment (kaustubhhiware) [reply notifyre.sh]: @athityakumar Weird, you never mentioned earlier.

Why don't you add an OSX alternative then, like before? (that OS check =="DARWIN") [If that is possible]

File: notifyre.sh (hunk 2 above)
Comment (kaustubhhiware) [reply notifyre.sh]: You can remove the` $?` part, it was merely for testing purposes. @athityakumar improved upon that by denoting the status by simple ticks and crosses.

Comment (a7r3) [reply notifyre.sh]: Good. I'm fine with it :+1:

File: notifyre.sh (hunk 1 above)
Comment (a7r3) [reply notifyre.sh]: @kaustubhhiware @athityakumar 

For now, the fallback i.e. `echo -e "\a"` should work on Mac Systems (Googled it)

Till then, you may find a workaround for playing custom notification files

**Comment by a7r3** [comment]
@kaustubhhiware Done

**Pull request merged**